field-descriptions.md

```diff
@@ -0,0 +1,186 @@
+# Description of configuration parameters
+
+This topic contains descriptions of the `lstreamed_default.conf` file parameters.  The `lstreamed_default.conf` parameters are used for  endpoint specific configurations.
+
+-  `ServerUrl`: Specifies the address of the server.
+
+    The protocol can be Kafka for Apache Kafka, HTTPs for Splunk and ElasticSearch, and HTTP for Zipkin.
+
+    The following are examples of how to specify the server URL for different end points:
+
+    Kafka:
+
+            kafka-server:9992
+            1.2.3.4:10000
+    Splunk:
+
+            http://splunk-server:80
+            https://splunk-server:443
+            http://1.2.3.4:1000
+            https://5.6.7.8:2000
+
+    ElasticSearch:
+
+            http://elastic-server:80
+            https://elastic-server:443
+            http://9.8.7.1:80
+            https://1.2.3.5:3000
+    Zipkin:
+
+            http://zipkin-server:80
+            http://1.2.3.4:80
+
+-  `KafkaTopic`:
+
+    Specifies the topic on a Kafka cluster for sending the transaction records. The default value is HTTP.
+
+-  `DataFormat`:
+
+    Specifies the format of the data sent over to Kafka. The values can be either JSON or AVRO.
+    The default data format is AVRO.
+
+-  `MaxConnections`:
+
+    Specifies the maximum parallel TCP connections to an endpoint.
+    The default value is 64.
+
+-  `FileSizeMax`:
+
+    For Kafka endpoint, AVRO files are created and stored on the disk before they get pushed to the endpoint.
+    This parameter specifies the size of each such file in Kibibyte (KiB). A file can contain multiple transaction records. The default value is 48 KiB.
+
+-  `RecordType`:
+
+    Specifies the types of records you want to export:
```

Comment:
ah can you please change this to :-  (i think this somehow got leaked)

`
HTTP
TCP
SWG
VPN
ICA
APPFW
BOT
VIDEOPT
BURST_CQA
SLA
MONGO
MQTT
`

Basically, remove `BOT_LE BOT_KM AS_FDI NGN`

Reply:
Removed.